php_env/PHP_8.1/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM php:8.1-cli
+
+# Update and import specific required librairies
+
+RUN apt-get update && apt-get install -y apt-utils
+RUN apt-get install -y \
+	unzip \
+	libicu-dev \
+    libonig-dev \
+	gcc \
+	wget \
+	zlib1g-dev \
+    libzip-dev
+
+# Parametrize PHP
+
+RUN docker-php-ext-install mbstring
+RUN docker-php-ext-install intl
+RUN docker-php-ext-install zip
+RUN docker-php-ext-install pcntl
+RUN docker-php-source delete
+
+# Install composer
+COPY --from=composer:2 /usr/bin/composer /usr/bin/composer
```

Comment:
Why is this step different from the other versions?

Reply:
Would you please check the screenshot?
![Screenshot From 2024-12-12 00-41-14](https://github.com/user-attachments/assets/8fe7531a-8be8-4da8-bc28-711ac24d17c7)

Reply:
@iulian03 this is the recommended version by composer
```